### ISSUE TITLE
[QUESTION] The location flag is not working ... 

### ISSUE THREAD

**Issue by 92sherlock** [twintproject/twint#968]
# Issue Template
Please use this template!

### Initial Check
> If the issue is a request please specify that it is a request in the title (Example: [REQUEST] more features). If this is a question regarding 'twint' please specify that it's a question in the title (Example: [QUESTION] What is x?). Please **only** submit issues related to 'twint'. Thanks.

>Make sure you've checked the following:

- [] Python version is 3.6;
- [] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran
>Please provide the _exact_ command ran including the username/search/code so I may reproduce the issue.

### Description of Issue
>Please use **as much detail as possible.**

### Environment Details
>Using Windows, Linux? What OS version? Running this in Anaconda? Jupyter Notebook? Terminal?

**Comment by himanshudabas** [comment]
Yes it's not working because it hasn't been ported to the new endpoints yet.

It will take some time to port it to the new endpoints.

**Comment by 92sherlock** [comment]
Oh okay. How long do you think it will take and is there an alternative.
Because I wish to have a geomap of my tweets.

**Comment by himanshudabas** [comment]
1. If I am not wrong, `Location` tries to find the location of the `User` of a _Tweet_.
So if you search for a query with `Location` flag enabled It will try to find the location of each _Tweet's Author_ (user who wrote that tweet).
Assuming that you get a total of 10,000 tweets for your search query, then _twint_ will have to find the location of each _unique author_ from the list of 10,000 tweets. If let's say around 8,000 Users tweeted these 10,000 tweets, then _twint_ will try to fetch the location of these 8,000 accounts.

2. > Because I wish to have a geomap of my tweets.

    But this sounds to me like you want to know where the _user_ actually tweeted from. Which I think are already available in the search results when it is saved to a `CSV`. In the csv you'll find a field called `geo` which contains the _Latitude & Longitude_ of where the tweet was tweeted from.

    Also currently there is another field in the `CSV` called _place_ which contains the same things as _geo_ but will be fixed in future patch.

    _Note: `geo` field in _CSV_ will not always be present, because most people don't tweet with GPS permission allowed to _Twitter_. so expect a lot of empty cells in your CSV._

So, I think If I am understanding you correctly 2nd option is what you need. Although If you need the 1st option then it's not yet implemented in the current version of twint. It might take a few weeks to implement it. No promises.

**Comment by 92sherlock** [comment]
Thank you for quick response Dabas. That's The problem I keep getting an
empty geo field for most of the tweets. I was thinking it should only
present the tweets with non empty geo field.

**Comment by benaqlan** [comment]
the same problem I can't get the location of tweets

**Comment by benaqlan** [comment]
but the geo field is empty in my CSV

**Comment by Mehul1604** [comment]
Hey , I am having the same issue. Turning the --location flag on just either makes it not do anything or it just finished without scraping anything.
Any fix to this??

**Comment by m-gaster** [comment]
Hi, any update to this issue? I'm still experiencing the same problem as Mehul

**Comment by rahmanusta** [comment]
Seems it is not working
